components/brave_wallet_ui/components/desktop/views/accounts/account.tsx

```diff
@@ -65,12 +65,14 @@ import { AccountButtonOptions } from '../../../../options/account-list-button-op
 import { useScrollIntoView } from '../../../../common/hooks/use-scroll-into-view'
 import {
   useGetVisibleNetworksQuery,
-  useGetUserTokensRegistryQuery
+  useGetUserTokensRegistryQuery,
+  useGetTransactionsQuery
 } from '../../../../common/slices/api.slice'
 import { useMultiChainSellAssets } from '../../../../common/hooks/use-multi-chain-sell-assets'
 
 // Actions
 import { AccountsTabActions } from '../../../../page/reducers/accounts-tab-reducer'
+import { skipToken } from '@reduxjs/toolkit/dist/query'
```

Comment:
Is this the right import? It should be the following according to the docs:

```ts
import { skipToken } from '@reduxjs/toolkit/query/react'
```

Reply:
Good question. They are the same. I updated the import to match the docs to avoid confusion.